src/popup/popup.css

```diff
@@ -60,12 +67,13 @@ button#stopMonitorBtn:hover {
 }
 
 .success {
-  background-color: #12bc00;
+  background-color: rgb(9, 92, 9);
```

Comment:
There is colors.css file with CSS variables for the colors from the photon style guide, we should include that stylesheet file and replace the colors used in the popup css with CSS variables (not just the colors being changed in this PR, also the other already being used).

The colors.css file is currently in the `src/activitylog/` directory because at the moment it is only used in the activitylog.html, we may move it in a shared stylesheet directory now that the popup should also be using it.